packages/search-adapter/src/empathy/__tests__/adapter/empathy-adapter-search.spec.ts

```diff
@@ -1,4 +1,5 @@
-import { BannerSchema, FacetSchema, PromotedSchema, RedirectionSchema, ResultSchema, TaggingSchema } from '@empathyco/x-types/schemas';
+import { BannerSchema, PromotedSchema, RedirectionSchema, ResultSchema, TaggingSchema } from '@empathyco/x-types/schemas';
```

Comment:
I don't understand this also. I can import all of these from the same package, that would be the `x-types-next`. But in the `package.json` of the adapter I can only find the `x-types` (not the next one). Should I change the `package.json` to get the new types and import all them from there?

Reply:
I'll try to explain myself better:

> You should not import from a package that is not defined as a dependency in the `package.json`

This is just a general rule. You made all imports from `@empathyco/x-types-next`, but there is no `@empathyco/x-types-next` entry at `packages/search-adapter/package.json` `dependencies` or `devDependencies` property.  This only works because it is searching in the parent `node_modules`, and lerna is hoisting the dependencies, but you should never do this.

> In this case you want the adapter to pick the search types of the mono-repo, there is no need to use two versions of search types here.

Instead of importing `@empathyco/x-types-next`, you just have to use the repository `x-types` package. Just like it was before. To do so just modify the `packages/search-adapter/package.json` `dependencies` entry with the following entry: `"@empathyco/x-types": "^10.0.0-alpha.7"`. Because this version matches the one defined in the `packages/search-types/package.json`, lerna will link it when bootstraping, and you can do all imports from `@empathyco/x-types` instead of `@empathyco/x-types-next`.

> Then in x-components you will have to lock the adapter version to the last released one, because it is not yet ready for this new adapter.

Because you are not going to update x-components to support the new adapter, you will have to lock its version from npm. To do so, modify `packages/x-components/package.json` `dependencies` with this entry: `"@empathyco/x-adapter": "npm:@empathyco/x-adapter@^6.2.0-alpha.8"`